lib/domain/model/extensions/mime_type_extension.dart

```diff
@@ -43,20 +43,26 @@ extension MediaTypeExtension on Event {
       'AttachmentExtension::getIcon(): mediaType: $mimeType || fileType: $fileType',
     );
     if (mimeType?.isEmpty == true || fileType == null) {
-      return ImagePaths.icFileUnKnow;
+      return ImagePaths.icFileUnknown;
     }
-    if (isDocFile()) {
-      return ImagePaths.icFileDocx;
-    } else if (isExcelFile()) {
-      return ImagePaths.icFileXlsx;
-    } else if (isPowerPointFile()) {
-      return ImagePaths.icFilePptx;
-    } else if (isPdfFile()) {
-      return ImagePaths.icFilePdf;
-    } else if (isZipFile()) {
-      return ImagePaths.icFileZip;
+
+    switch (getSupportedPreviewFileType()) {
+      case SupportedPreviewFileTypesEnum.image:
+        return ImagePaths.icFileUnknown;
```

Comment:
why `image` return `unknown`?

Reply:
Images have not icon since they have thumbnails instead, it is the same logic as before I just refactored it to use switch case